.run/jules_run.run.xml

```diff
@@ -0,0 +1,24 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
These files are intellij run configs. We experimented with sharing them. Should probably be removed from the repo.